packages/pg-pool/index.js

```diff
@@ -248,6 +251,19 @@ class Pool extends EventEmitter {
       } else {
         this.log('new client connected')
 
+        if (0 !== this.options.maxLifetimeSeconds) {
```

Comment:
minor style consistency

```suggestion
        if (this.options.maxLifetimeSeconds !== 0) {
```